INTERESTED_DEVELOPERS.md

```diff
@@ -0,0 +1,25 @@
+# Interested Developers:
+This file will help us track people that are interested in taking decisions of the specification of GraphQL over HTTP until merged to the main repository located at https://github.com/facebook/graphql
```

Comment:
I think GraphQL specification repo is not the right place for this specification since GraphQL should stay language agnostic.

My final goal is to move it under `github.com/graphql/graphql-over-http` so it would leave as a separate spec but under official GraphQL organization.

Reply:
LGTM

Reply:
This file will help us track people that are interested in taking decisions of the specification of GraphQL over HTTP until migrated to a official repository of the https://github.com/graphql organization.

Reply:
Sounds good 👍 Thank you.